docs/installation/eks.md

```diff
@@ -0,0 +1,163 @@
+# Install on EKS
+
+<details markdown="1">
+<summary>TIP: Make sure you have enough GPU quota in your AWS account.</summary>
+
+The default quotas for GPU instances are often 0. You will need to request a quota increase for the GPU instances you want to use.
+
+The following quotas may require an increase if you wish to use GPUs in your EKS cluster:
+- All G and VT Spot Instance Requests
+- All P5 Spot Instance Requests
+- All P4, P3 and P2 Spot Instance Requests
+- Running Dedicated p4d Hosts
+
+</details>
+
+## 1. Create EKS cluster with Karpenter
+
+Configuration used throughout this guide:
```

Comment:
Example configuration:

Reply:
I rephrased, but it's not just example config, those environment variables need to be set. Rephrased to:

Set the environment variables used throughout this guide: